keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -551,16 +600,22 @@ Below are some examples to consider, in addition to the aforementioned [maturity
 -->
 #### Alpha
 
-- Feature implemented behind a feature flag
+- Feature implemented behind a feature gate
 - Integration tests added for all relevant verbs (POST, PUT, and PATCH)
 
-<!--
 #### Beta
 
-- Gather feedback from developers and surveys
-- Complete features A, B, C
-- Additional tests are in Testgrid and linked in KEP
+- [  ]kubectl validate flag offers ability to perform server-side validation
+- [ ] endpoints handler unit testing of field validation
+- [ ] customresource handler unit testing of field validation
```

Comment:
I thought we already had these tests

Reply:
We have integration tests for each endpoint via [field_validation_test.go](https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/field_validation_test.go) and we have unit test coverage of the various field validation algorithms we touched (i.e structural pruning and unstructured converter), but this is referring to testing just the handler behavior independently for requests with invalid fields (i.e. [apiserver_test.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go) and [customresource_handler_test.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler_test.go).

Are these the tests you though we already have

Reply:
yeah, that was what I was thinking of